src/irmin-mirage/git/irmin_mirage_git.ml

```diff
@@ -261,30 +281,59 @@ module KV_RW (G : Irmin_git.G) (C : Mirage_clock.PCLOCK) = struct
   let info t op = Info.f ~author:(t.author ()) "%s" (t.msg op)
   let path = RO.path
 
+  let ( >?= ) r f =
```

Comment:
Nit: Do we need this? And if we need this should it maybe go to `irmin/import.ml` which defines some other Lwt helper syntax?

Reply:
The motivation for this (and `lwt_ok`, although I think this infix operator is more helpful) was to remove some of the noise in code that needs to do type coercion for `write_error` and hopefully clarify the main flow of the code. You can see one example in the removed lines 267-270 below. I needed this type coercion in the new code and doing a lot of `match` or `function` just to coerce the error type was quite noisy. :sweat_smile: 

Unfortunately, it wouldn't fit in `irmin/import.ml` because it is specific to this code.